bin/dwh-migration-dumper

```diff
@@ -7,4 +7,7 @@ if [ ! -x "$BIN" ] ; then
   (cd "${BASE_DIR}" && ./gradlew --parallel :dumper:app:installDist)
 fi
 
-exec "$BIN" "$@"
+# "ALL-UNNAMED" option is required as a workaround for Snowflake JDBC Driver error.
+_JAVA_OPTIONS="--add-opens=java.base/java.nio=ALL-UNNAMED"
+
+exec env _JAVA_OPTIONS="$_JAVA_OPTIONS" "$BIN" "$@"
```

Comment:
Do you know, if the script would also work without changing this line?

For example,
```
export _JAVA_OPTIONS="--add-opens=java.base/java.nio=ALL-UNNAMED"
exec "$BIN" "$@"
```

Reply:
I tried and it works.

However, the `env` method is recommended by the official Arrow site: https://arrow.apache.org/docs/java/install.html#java-compatibility

Reply:
After reading a bit more about `env` and `export`, I think they are unrelated to Arrow. The `export` command is a bash built-in and `env` is a command. They differ, but for the purpose of this script they both do the same thing (unless this script is not executed in Bash - due to `/bin/sh` in the first line, which is not necessarily Bash). If it was guaranteed that the script is executed in Bash (we should change the shebang to `/bin/bash`), I would prefer `export` so as 1. not to mix too many things on a single line and 2. keep the code short.

```
export _JAVA_OPTIONS="$_JAVA_OPTIONS --add-opens=java.base/java.nio=ALL-UNNAMED"
exec "$BIN" "$@"
```

However, if we leave `/bin/sh`, then `env` is probably the most portable solution.

https://askubuntu.com/questions/205688/whats-the-difference-between-set-export-and-env-and-when-should-i-use-each

https://unix.stackexchange.com/questions/368944/what-is-the-difference-between-env-setenv-export-and-when-to-use